OneCommunityGlobal/HighestGoodNetworkApp
# Imran added General Section to Permissions

# Description
A hotfix to move the one at "See All Users in Dashboard and Leaderboard" Permission to a new section at the top called “General” and the “Edit Team 4-Digit Codes” to the Reports section.

## Related PRS (if any):
N/A

## Main changes explained:
- updated the PermissionsConst file
…

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. Clear site data/cache
4. log as admin user
5. go to Other Links -> Permissions Management -> Manage User Permissions
## Screenshots or videos of changes:

## Note:
Include the information the reviewers need to know.
